MirrorProvider/MirrorProvider.Linux/LinuxFileSystemVirtualizer.cs

```diff
@@ -219,7 +219,7 @@ private bool TryGetSymLinkTarget(string relativePath, out string symLinkTarget)
 
             const ulong BufSize = 4096;
             byte[] targetBuffer = new byte[BufSize];
-            long bytesRead = ReadLink(fullPathInMirror, targetBuffer, BufSize);
+            long bytesRead = Syscall.readlink(fullPathInMirror, targetBuffer);
```

Comment:
That's nice!

Reply:
I pulled `Mono.Posix` in in https://github.com/Microsoft/VFSForGit/pull/800, and it would be remiss not to use it where appropriate!